### ISSUE TITLE
Calculating point-to-surface (p2s) distance

### ISSUE THREAD

**Issue by penincillin** [vchoutas/expose#19]
Hi,

Thanks for sharing this wonderful code.

I have a question about how to calculate the point-to-surface (p2s) error used in Table 3.

Would you mind give me some clues on this ?

Thanks !

**Comment by penincillin** [comment]
Oh, I've find them in the code.

**Comment by vchoutas** [comment]
@penincillin The code in the repo only has the vertex-to-vertex error computation.
For the point-to-surface metric, we first do a rigid alignment between the ground truth and estimated mesh and then compute the distance between every vertex on the ground truth mesh to the surface of the estimated mesh. Here by ground truth mesh I mean the SMPL-X registration.
Unfortunately, the code for this is not public, I'll try to see how we can release this.

**Comment by booker-max** [comment]
Do you have the training code for this paper? Can you share it with me? This is my email address 1654617276wj@gmail.com，Thank you very much.！
